dockstore-integration-testing/src/test/java/io/dockstore/client/cli/NotificationIT.java

```diff
@@ -45,6 +50,15 @@ private Notification anotherTestNotification() {
         return notification;
     }
 
+    private Notification longNotification() throws IOException {
+        Notification notification = new Notification();
+        String message = Files.readString(Paths.get(ResourceHelpers.resourceFilePath("longNotification.txt")));
```

Comment:
I know you just changed this, and I overlooked it in my first review, but I think it would be better to generate the string right here. Java 11 has a feature has a feature `Sting.repeat`. If you do it this way:

1. You know exactly how long the string is. How long is longNotification.txt? I have to open it an editor or run some tool that counts chars. At a glance, I don't know if you're really over testing the limit or not.
2. You could easily test different inputs, up to the limit, just over the limit, etc.

Something like this:

```
String message = "a".repeat(1024);
```

You could make 1024 a parameter on line 53, so then you test it with exactly 1024, then test again with 1025.

You could even make the length a parameter on line 53.